pogo/session.py

```diff
@@ -561,6 +562,26 @@ def setFavoritePokemon(self, pokemon, is_favorite):
         # Return Everything
         return self._state.favoritePokemon
 
+    # Upgrade a Pokemon's CP
+    def upgradePokemon(self, pokemon)
```

Comment:
Fixed.

On Jul 29, 2016 16:03, "Nathan Bakirci" <notifications@github.com> wrote:

> In pogo/session.py
> <https://github.com/rubenvereecken/pokemongo-api/pull/53#discussion_r72872664>
> :
>
> > @@ -561,6 +562,26 @@ def setFavoritePokemon(self, pokemon, is_favorite):
> >          # Return Everything
> >          return self._state.favoritePokemon
> >
> > +    # Upgrade a Pokemon's CP
> > +    def upgradePokemon(self, pokemon)
>
> colon?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rubenvereecken/pokemongo-api/pull/53/files/52a23dfc165b2c1db5021483963b61ea4bf3a155#r72872664>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APgjsMap2fPYbs_uVqgz2ExW8TRB_5qhks5qaobVgaJpZM4JYohu>
> .
>